docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature: `1201` Adds warning to password change when premium sync is enabled.
```

Comment:
Can you also maybe change the text you put in the changelog both in this and the others? It should not be the same as the PR title. Think how it will read when it will be in the release notes.

Example: If a user attempts to change their password when premium sync is enabled a warning is now displayed.

Reply:
Well, in my mind, it reads "*This version/this release*, adds (a) warning to password change when premium sync is enabled.

Reply:
Yeah I understand. We should have a common pattern though. I hope you understand. When we make a release we just copy paste those. So they need follow a consistent writing style.